parler/models.py

```diff
@@ -1057,6 +1157,8 @@ def __init__(self, shared_model, translations_model, related_name):
         self.shared_model = shared_model
         self.model = translations_model
         self.rel_name = related_name
+        self.rel_name_active = None
+        self.rel_name_default = None
```

Comment:
What are `rel_name_default` and `rel_name_active` doing?

Reply:
This the names for new 2 attributes to the main model with virtual composite foreign key. Refer to translations in default and active languages.